### PR TITLE
Add landing page for the project

### PR DESCRIPTION
**Pull request by arjavibahety**
Todo:
- [x] Add Michaela's contact details
- [ ] Finish contact us form after discussing with the team

### CODE REVIEW

File: webproject/src/main/webapp/index.html [hunk 1]
```diff
@@ -225,23 +219,20 @@ <h2 class="section-title mb-3">Our Team</h2>
                   alt="Image"
                   class="w-100 img-fluid mb-3 shadow team-member"
                   src="images/Arjavi.jpg"
-                  alt=""
-                />
+                  alt=""/>
```
Comment (lyskevin) [inline webproject/src/main/webapp/index.html]: Do we want to have no space (as seen here) between the ending tag and the closing /> brace or should we have one spacing (as seen in some of the code above e.g. the new line 8)?

Comment (arjavibahety) [reply webproject/src/main/webapp/index.html]: I think it is better to have one spacing. Thanks for pointing it out!

File: webproject/src/main/webapp/index.html [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+            <div class="col-lg-6 team-member-container">
+              <div class="team-member-img">
+                <img
+                  alt="Image"
+                  class="w-100 img-fluid mb-3 shadow team-member"
+                  src="images/Arjavi.jpg" />
+              </div>
+              <div class="team-member-details">
+                <h4>
+                  Bahety Arjavi
```
Comment (kevinluvian) [inline webproject/src/main/webapp/index.html]: Michaela missing? haha

**Review by kevinluvian — CHANGES_REQUESTED** [review]

**Review by kevinluvian — APPROVED** [review]
LGTM, we can do the contact form on other PR

**Pull request merged**